README.md

```diff
@@ -112,6 +112,7 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
 
 * [osmcha](https://osmcha.org) - Detector for suspicious changesets. ([Source Code](https://github.com/mapbox/osmcha-frontend) / [Wiki](https://wiki.openstreetmap.org/wiki/OSMCha))
 * [achavi](https://overpass-api.de/achavi/) - Augmented change viewer. ([Source Code](https://github.com/nrenner/achavi/) / [Wiki](https://wiki.openstreetmap.org/wiki/Achavi))
+* [latest-changes](https://rene78.github.io/latest-changes/#2/15.0/-15.0) - Visualize latest OSM changesets within a certain geographic boundary. Includes simple vandalism checker. ([Source Code](https://github.com/rene78/latest-changes/))
```

Comment:
I agree with the recategorization, but please move this to the bottom of the category as the categories are supposed to be sorted by date added and adding in the middle (even if just for a move) would kinda undermine that 😛